include/gatekeeper_net.h

```diff
@@ -32,6 +34,7 @@ struct net_config {
 	uint32_t		num_ports;
 	uint32_t		numa_nodes;
 	struct rte_mempool 	**gatekeeper_pktmbuf_pool;
+	uint16_t		queues[RTE_MAX_LCORE];
```

Comment:
We need four arrays to accommodate queues per tx and rx and per interface.

The motivation for this is that the blocks don't have a uniform queue demand. The GK-GT unit for example requires a rx and tx queue from the back interface, but none from the front interface.